lib/puppetserver/ca/generate_action.rb

```diff
@@ -94,10 +95,20 @@ def generate_root_and_intermediate_ca(settings, signing_digest)
           [settings[:cacrl], [int_crl, root_crl]]
         ]
 
+        errors = FileUtilities::check_for_existing_files(file_properties.map { |prop| prop.first })
```

Comment:
should this be 
```
FileUtilities.check_for_existing_files
```

Reply:
Yeah, `::` used to be idiomatic in older Ruby for any class access (constant and method) but I think that `.` has become idiomatic for any method access regardless of instance vs class.